daq_lib.py

```diff
@@ -788,8 +790,22 @@ def center_on_click(x,y,fovx,fovy,source="screen",maglevel=0,jog=0): #maglevel=0
       setPvDesc("image_X_scaleMM",float(fovx))
       setPvDesc("image_Y_scaleMM",float(fovy))
 
+  # The following is intended to allow basic axis changes for standard off-axis camera views
+  # "beam" - camera view is directly in line with beam
+  # "above" - camera is looking down from directly above
+  # "below" - camera is looking up from directly below
+  if (viewangle=="beam"):
+    correctedX = x * daq_utils.unitScaling
+    correctedY = y * daq_utils.unitScaling
+  if (viewangle=="above"):
+    correctedX = x * daq_utils.unitScaling
+    correctedZ = y * daq_utils.unitScaling
+  if (viewangle=="below"):
+    correctedX = x * daq_utils.unitScaling
+    correctedZ = -y * daq_utils.unitScaling
+  
```

Comment:
code like this should handle the possibility of undefined variables - it really should be `elif` for the latter 2 and an exception if `viewangle` is not a valid value

While I personally would rather have the axis definitions done with individual definitions, having groupings like this can help the code be more readable. we should consider extracting those names into config_params in the future